subsys/bluetooth/common/Kconfig

```diff
@@ -103,6 +103,21 @@ config BT_DEBUG_HCI_DRIVER
 	  Bluetooth HCI driver, including the Controller-side HCI layer
 	  when included in the build.
 
+config SYS_LOG_BT_HCI_DRIVER_LEVEL
```

Comment:
This will conflict with #10264. Any chance you could just remove it from the PR?

Reply:
Or to put it another way: at least follow the convention of existing HCI drivers and logging instead of introducing this piece that will anyway be removed once #10264 gets merged.

Reply:
I marked this PR as DNM because I was aware of a couple of different activities that would impact various things including the logging changes. At the time of this edit, there wasn't a way to just enable logging for the HCI driver since it appeared to be tied to the BLE logging so I added this. There are some internal NXP things we are trying to work out for hosting the library which will likely take longer then the #10264 PR so, yes, I will move this to the new system when it is ready.

Reply:
By the way, I did start making changes to align with the new zephyr logging system going in but stopped to wait instead on when the BT subsystem catches up.